### ISSUE TITLE
Ramps at the end of (otherwise) dead ends

### ISSUE THREAD

**Issue by MoKob** [Project-OSRM/osrm-backend#4210]
![screen shot 2017-06-28 at 14 29 33 2](https://user-images.githubusercontent.com/12932279/27636995-5253e954-5c0e-11e7-90d7-e504e7e9b242.png)

[osrm-link](http://map.project-osrm.org/?z=18&center=38.900020%2C-77.054576&loc=38.900978%2C-77.056067&loc=38.899579%2C-77.054232&hl=en&alt=0)

Some ramps can be simply at the end of an otherwise dead end.
Announcing `take the ramp` (in this case even to the right) does not feel like a reasonable announcement here, especially since the two turns are so close to each other.

To me, personally, it is unclear what would be a good announcement for these combination of turns, but I feel that we probably cannot do much better, except for ignoring the `direction_modifier` to turning onto the ramp.

**Comment by emiltin** [comment]
continue on ramp? or just omit it, since there is no choice of route involved

**Comment by TheMarex** [comment]
Yeah I would say the expected behavior here would be that we don't emit an instruction since there is no action needed on part of the driver.
